libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Selectors/Microsoft.ConditionalSelector.schema

```diff
@@ -0,0 +1,19 @@
+﻿{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/botbuilder-tools/SchemaGen/packages/DialogSchema/src/dialogSchema.schema",
+    "$role": "unionType(Microsoft.IRuleSelector)",
+    "title": "Use a rule selector based on a condition",
+    "description": "Use a rule selector based on a condition",
+    "type": "object",
+    "properties": {
+        "condition": {
+            "$type": "Microsoft.IExpression"
+        },
+        "iftrue": {
+            "$type": "Microsoft.IRuleSelector"
+        },
+        "iffalse": {
```

Comment:
>iffalse [](start = 9, length = 7)

ifFalse

Reply:
Changed both.

Reply:
what about "if", "else" ?

---
In reply to: [274716751](https://github.com/Microsoft/botbuilder-dotnet/pull/1655#discussion_r274716751) [](ancestors = 274716751)